src/spec/usecase/usecase_a.js

```diff
@@ -0,0 +1,97 @@
+export default (Passable) => {
+
+    const response = Passable('case_a', (pass, enforce) => {
+        pass('field_1', 'should be a string of 5 chars', () => {
+            enforce('hello').allOf({
+                sizeEquals: 5,
+                isString: true
+            });
+        });
+
+        pass('field_2', 'must be a number smaller than 90', () => {
+            enforce(99).allOf({
+                smallerThan: 90,
+                isNumber: true
+            });
+        });
+
+        pass('field_3', 'must be either "a" or "b"', () => {
+            enforce('a').allOf({
+                inside: ['a', 'b']
+            });
+        });
+
+        pass('field_4', 'should be either "a" or "b"', 'warn', () => {
+            enforce('c').allOf({
+                inside: ['a', 'b']
+            });
+        });
+
+        pass('field_5', 'Must either be a number or a string. Always smaller than 5', () => {
+            enforce('log').anyOf({
+                isNumber: true,
+                isString: true
+            }).allOf({
+                smallerThan: 5
+            });
+        });
+
+        pass('field_5', 'Must not be larger than 3', () => {
```

Comment:
Are you intentionally using `field_5` twice?
Same for the other cases.

Reply:
Yes, Passable allows writing multiple tests for the same field. Some of which may fail and some may pass. Here I am testing its ability to do so correctly.